wizard.py

```
@@ -314,19 +314,21 @@ def setup_git_hooks():
             import shutil
+            # Ensure config/ directory exists
+            config_file.parent.mkdir(parents=True, exist_ok=True)
             shutil.copy(config_template, config_file)
-            console.print("✅ [green]Created config.yml from template[/green]")
+            console.print("✅ [green]Created config/config.yml from template[/green]")
         else:
-            console.print("⚠️  [yellow]config.yml.template not found, skipping config setup[/yellow]")
+            console.print("⚠️  [yellow]config/config.yml.template not found, skipping config setup[/yellow]")
     else:
-        console.print("ℹ️  [blue]config.yml already exists, keeping existing configuration[/blue]")
+        console.print("ℹ️  [blue]config/config.yml already exists, keeping existing configuration[/blue]")
```

Ruff (0.14.10)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)